addons/escoria-core/game/core-scripts/esc/commands/turn_to.gd

```diff
@@ -2,9 +2,13 @@
 # 
 # Turns object to face another object. 
 #
-# The wait parameter sets how long to wait for each intermediate angle. It
-# defaults to 0, meaning the turnaround is immediate.
-#  
+# **Parameters**
+#
+# - *object*: Global ID of the object to turn
+# - *object_to_face*: Global ID of the object to turn to
+# - *wait*: How long to wait for each intermediate angle. If set to 0 the
```

Comment:
It would be interesting to indicate the type (int, float?) and the duration type (seconds? milliseconds?)

Reply:
Fixed by Duncan